src/openrct2/world/Map.cpp

```diff
@@ -1197,7 +1197,8 @@ TileElement* tile_element_insert(const CoordsXYZ& loc, int32_t occupiedQuadrants
     newTileElement->SetLastForTile(isLastForTile);
     newTileElement->SetOccupiedQuadrants(occupiedQuadrants);
     newTileElement->SetClearanceZ(loc.z);
-    std::memset(&newTileElement->pad_04, 0, sizeof(newTileElement->pad_04));
+    newTileElement->owner = 0;
```

Comment:
Might be worth to introduce a constant for this where 0 is undefined, ownerless should be perhaps the maximum possible value, this ensures we are able to tell the difference for when we actually use this field for compatibility with saves generated before.

Reply:
I suppose 0 could mean "unowned" and 1 could mean "park 1". The advantage of this would be that it is trivial to increase the limit beyond 15 later on.